samples/api-client/mqtt_example/noxfile.py

```diff
@@ -0,0 +1,266 @@
+# Copyright 2019 Google LLC
```

Comment:
```suggestion
# Copyright 2021 Google LLC
```

Reply:
Please don't forget the year update here!

Reply:
done.